libraries/Microsoft.Bot.Builder.Ai.Translation/PostProcessor/TranslatedDocument.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Bot.Builder.Ai.Translation
+{
+    /// <summary>
+    /// Translated document is the data object holding all information of the translator module output on an input string
+    /// </summary>
+    public class TranslatedDocument
+    {
+        private string sourceMessage;
```

Comment:
member variables should have a "_" on them.

Reply:
Most of these variables are not needed. They can be "implied" form the properties. 

public string SourceMessage { get ;set; }